cloudfour/cloudfour.com-patterns
# Outdent Group Blocks with Backgrounds

## Overview

This PR updates the the styles for Gutenberg "group" blocks with a
background color set to apply our standard padding, plus outdent the
block a bit to keep the group content aligned with the page content.

## Screenshots

<img width="600" alt="Screen Shot 2022-06-13 at 1 45 39 PM" src="https://user-images.githubusercontent.com/257309/173442274-f9a98c57-9fea-42b3-a8fb-9d935dc0a892.png">

## Testing

1. On the preview deploy, review the "Group block" story
1. Use the arguments panel to apply a background
2. Confirm that the group's content is properly aligned with the page content.

---

- Fixes #1824
